### PR TITLE
[zkp-iden3] Add zkp-iden3 package

### PR DESCRIPTION
**Pull request by ardier16**

### CODE REVIEW

File: packages/zkp-iden3/README.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+const issuerId = DID.idFromDID(did);
+
+const issuerHexId = `0x0${issuerId.bigInt().toString(16)}`;
+
+const stateData = await loadDataFromRarimoCore<GetStateInfoResponse>(
+  `/rarimo/rarimo-core/identity/state/${issuerHexId}`,
+  chainInfo.rarimoApiUrl,
+);
+// const merkleProof = await loadDataFromRarimoCore<MerkleProof>(
```
Comment (ihordiachenko) [inline packages/zkp-iden3/README.md]: Uncomment or remove

File: packages/zkp-iden3/package.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
+{
+  "name": "@rarimo/zkp-iden3",
+  "version": "2.1.0-rc.3",
```
Comment (ihordiachenko) [inline packages/zkp-iden3/package.json]: I think this package should be versioned independently of the snap stuff, because it doesn't depend on it

Comment (ihordiachenko) [reply packages/zkp-iden3/package.json]: And a separate changelog

Comment (lukachi) [reply packages/zkp-iden3/package.json]: it depends on connector types and helpers

Comment (ihordiachenko) [reply packages/zkp-iden3/package.json]: Anyway, it breaks semver for the package consumers. Let's merge it for now, but we need to refactor it in the future

**Review by ihordiachenko — APPROVED** [review]

**Pull request merged**